bodhi/server/util.py

```diff
@@ -982,26 +955,22 @@ def sorted_updates(updates):
             with a multicall.
     """
     builds = defaultdict(set)
-    build_to_update = {}
     sync, async = [], []
     for update in updates:
         for build in update.builds:
-            n, v, r = get_nvr(build.nvr)
-            builds[n].add(build.nvr)
-            build_to_update[build.nvr] = update
-    for package in builds:
+            builds[build.nvr_name].add(build)
+    # The sorted here is so we actually have a way to test this
+    # Otherwise, we would be depending on the way Python orders dict keys
+    for package in sorted(builds.keys()):
         if len(builds[package]) > 1:
-            log.info('Found multiple %s packages' % package)
             log.debug(builds[package])
             for build in sorted_builds(builds[package])[::-1]:
-                update = build_to_update[build]
```

Comment:
Hahah what the heck was this here for? Nice of you to fix that to a simple ```build.update``` ☺

Reply:
This was because in the original code, it was dealing with the nvr strings, not the actual build objects.